protocol/src/private_id/company.rs

```diff
@@ -80,9 +95,9 @@ impl CompanyPrivateIdProtocol for CompanyPrivateId {
                 .clone()
                 .write()
                 .map(|mut d| {
-                    let t = timer::Timer::new_silent("Load e_company");
+                    #[cfg(not(target_arch="wasm32"))]  let t = timer::Timer::new_silent("Load e_company");
```

Comment:
I removed all these directives and instead made a small modification to `common/src/timer.rs` to actually support your common::timer::Timer class because it may be useful to keep (for use in wasm32).